tests/functional/adapter/test_custom_schema.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from dbt.tests.util import run_dbt, run_sql_with_adapter
+
+seed_csv = """
+id,name,date
+1,Easton,1981-05-20 06:46:51
+2,Lillian,1978-09-03 18:10:33
+3,Jeremiah,1982-03-11 03:59:51
+4,Nolan,1976-05-06 20:21:35
+""".lstrip()
+
+
+class TestCustomSchemaBase:
+    """
+    This test is meant to ensure that Trino table, view, incremental materialization
+    works as expected for custom schemas
+    """
+
+    # Setting __test__ to False, to avoid testing base class
+    __test__ = False
```

Comment:
This seems not how we do it in other classes. I think you shouldn't name your class `Test...`.